terraform/root/aws-dm-dev/main.tf

```diff
@@ -8,3 +8,9 @@ module "aws_env" {
   aws_main_account_id = "${var.aws_main_account_id}"
   aws_dev_account_id = "${var.aws_dev_account_id}"
 }
+
+module "s3-bucket" {
```

Comment:
Cool. My original thought was to just have an s3-bucket module which could then be pulled in by any of the environments and parameterised to suit, but as you can't reuse the same name I don't think it'd work. I think this bucket/policy is quite specific so I'll move it in.